src/Sql/Sql/TransparentDataEncryption/Cmdlet/RevalidateAzureRmSqlManagedInstanceTransparentDataEncryptionProtector.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Microsoft.Azure.Commands.Sql.TransparentDataEncryption.Model;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.Sql.TransparentDataEncryption.Cmdlet
+{
+    /// <summary>
+    /// Defines the Revalidate-AzSqlInstanceTransparentDataEncryptionProtector cmdlet
+    /// </summary>
+    [Cmdlet("Invoke", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "SqlInstanceTransparentDataEncryptionProtectorRevalidation", SupportsShouldProcess = true, DefaultParameterSetName = DefaultParameterSet)]
+    [OutputType(typeof(AzureRmSqlManagedInstanceTransparentDataEncryptionProtectorModel))]
+    public class RevalidateAzureRmSqlManagedInstanceTransparentDataEncryptionProtector : AzureSqlRmManagedInstanceTransparentDataEncryptionProtectorBase
+    {
+        /// <summary>
+        /// Defines whether it is ok to skip the revalidate Transparent Data Encryption protector request confirmation
+        /// </summary>
+        [Parameter(HelpMessage = "Skip confirmation message for performing the action")]
+        public SwitchParameter Force { get; set; }
```

Comment:
-Force here is unnecessary, if you add  SupportsShouldProcess, we will have -Confirm to allow to skip confirmation